### PR TITLE
Fix/remove app deletion

### PR DESCRIPTION
**Pull request by WRadoslaw**

### CODE REVIEW

**Review by drillprop — CHANGES_REQUESTED** [review]
1. Totally unrelated, but could you also fix a small typo in `cli/examples/content/CreateApp.json` and `cli/examples/content/UpdateApp.json` as part of this PR? You wrote "exmaple" instead of "example".  

2. You didn't remove `deleteApp` test from the `full.ts`

**Review by Lezek123 — COMMENTED** [review]
Please also remove:
- `deleteApp` CLI command
- References to `deleteApp` protobuf message in `/tests/network-tests/src/Api.ts`
- Reference to `deleteApp` CLI command in `/tests/network-tests/src/cli/joystream.ts`

File: tests/network-tests/src/Api.ts [hunk 1]
```diff
@@ -816,62 +816,6 @@ export class Api {
...
   }
 
-  async createApp(name: string, appMetadata: IAppMetadata, memberId: u64): Promise<ISubmittableResult> {
-    const account = await this.getMemberControllerAccount(memberId.toNumber())
-
-    if (!account) {
-      throw new Error('invalid account')
-    }
-    const meta = new MemberRemarked({
-      createApp: new CreateApp({
```
Comment (Lezek123) [inline tests/network-tests/src/Api.ts:826]: It looks like you forgot about removing imports from `@joystream/metadata-protobuf` (`CreateApp`, `DeleteApp`, `IAppMetadata`, `UpdateApp`), so the CI checks are failing: https://github.com/Joystream/joystream/actions/runs/4283418182/jobs/7458964301

**Review by Lezek123 — APPROVED** [review]
LGTM

**Pull request merged**